src/main/java/org/reaktivity/nukleus/kafka/internal/stream/KafkaMergedFactory.java

```diff
@@ -802,7 +802,7 @@ private static boolean hasProduceCapability(
         private final long defaultOffset;
         private final KafkaDeltaType deltaType;
 
-        private KafkaAge maximumAge;
+        private KafkaOffsetType maximumAge;
```

Comment:
Please rename `maximumAge` to `maximumOffset` now that we have changed the type from `KafkaAge` to `KafkaOffsetType`.